test/parameters_test.rb

```diff
@@ -34,6 +34,27 @@
       delocalized_params[:product][:price].must_equal '1299.99'
     end
 
+    it "delocalizes field-for type params based on the given options" do
+      params = parameters_class.new(
+          :product => {
+              variant_attributes: {
+                  "0" => { :released_on => '21. Mai 1986', :available_until => '25. Dezember 2013, 23:59 Uhr', :price => '1.299,99' },
+                  "1" => { :released_on => '21. Mai 1986', :available_until => '25. Dezember 2013, 23:59 Uhr', :price => '1.299,99' },
```

Comment:
Nitpick: For the sake of testing that we don't accidentally write wrong values, the second variant should have different values.